build/Targets/Signing.Imports.targets

```diff
@@ -76,5 +76,21 @@
       <DelaySign>true</DelaySign>
     </PropertyGroup>
   </Target>
+  
+   <!--
+    Workaround https://github.com/Microsoft/msbuild/issues/1490 (lack of support for public-signing satellites) by 
+    delay-signing satellite assemblies when main assembly is public-signed.
```

Comment:
It means that if you build with desktop msbuild and non-English, you have to have the skip verification that our preinstall script gives you. If you build with core msbuild (as build.cmd does) there's no impact as we will be able to public sign. In official builds, everything is always delay-signed in the build step and then sent to be full signed so there is no product impact.